Packs/BmcITSM/Integrations/BmcITSM/test_data/list_tickets_work_order.json

```diff
@@ -0,0 +1,226 @@
+                "WO Type Field 15 Label": ".",
+                "WO Type Field 16 Label": ".",
+                "WO Type Field 17 Label": ".",
+                "WO Type Field 18 Label": ".",
+                "WO Type Field 19 Label": ".",
+                "WO Type Field 20 Label": ".",
+                "WO Type Field 21 Label": ".",
+                "WO Type Field 22 Label": ".",
+                "WO Type Field 23 Label": ".",
+                "WO Type Field 24 Label": ".",
+                "WO Type Field 25 Label": ".",
+                "WO Type Field 26 Label": ".",
+                "WO Type Field 27 Label": ".",
+                "WO Type Field 28 Label": ".",
+                "WO Type Field 29 Label": ".",
+                "WO Type Field 30 Label": ".",
+                "z1D_WorkInfoSubmitter": null,
+                "AttachmentSourceFormName": null,
+                "AttachmentSourceGUID": null,
+                "AttachmentSubmitter": null,
+                "SRWorkInfoType": null,
+                "z1D_ConfirmGroup": null,
+                "CreatedFromBackEndSynchWI": null,
+                "WO Type Field 48": null,
+                "WO Type Field 49": null,
+                "WO Type Field 50": null,
+                "WO Type Field 51": null,
+                "WO Type Field 48 Label": ".",
+                "WO Type Field 49 Label": ".",
+                "WO Type Field 50 Label": ".",
+                "WO Type Field 51 Label": ".",
+                "Chat Session ID": null,
+                "Broker Vendor Name": null,
+                "NeedsAttentionCCS_Setting": "false",
+                "Automation Status": "Manual",
+                "RequestCreatedFromDWP": "No",
+                "DWP_SRID": null,
+                "DWP_SRInstanceID": null,
+                "WO Type Field 01 Label": "Username",
+                "WO Type Field 02 Label": "Sharefolder Name",
+                "WO Type Field 03 Label": "Access Type",
+                "WO Type Field 04 Label": ".",
+                "WO Type Field 05 Label": ".",
+                "WO Type Field 06 Label": ".",
+                "WO Type Field 07 Label": ".",
+                "WO Type Field 08 Label": ".",
+                "WO Type Field 09 Label": ".",
+                "z1D Char09": null,
+                "CI_DatasetId": "BMC.ASSET",
+                "CI_ReconId": null,
+                "Description": "New Share Folder Access",
+                "Location Company": "Calbro Services",
+                "Organization": "Information Technology",
+                "Support Organization": "IT Support",
+                "Support Group Name": "Service Desk",
+                "Last Name": "Agent",
+                "First Name": "Arthur",
+                "Middle Initial": null,
+                "VIP": "No",
+                "Chg Location Address": "Boeing Avenue 245 \r\nSchiphol-Rijk, Amsterdam  1119 PD \r\nNetherlands",
+                "Internet E-mail": "arthur_agent@calbro.com",
```

Comment:
change to a mail thats more obviously not important and remove from secret ignore